camunda-formio-tasklist-vue/src/services/Constants.ts

```diff
@@ -22,3 +22,10 @@ export const FORMLIST_FIELDS = [
     key: "operations" 
   }
 ];
+
+ const porjectUrl =  process.env.VUE_APP_FORM_IO_PROJECT_URL|| 'http://127.0.0.1:3000'
```

Comment:
Don't use env. It's against approach our component and can't used for npm package.